ExampleCode/C++/API/API.cpp

```diff
@@ -80,22 +81,38 @@ void iSENSE::set_contributor_key(string proj_key)
 }
 
 
-// Sets the email address to be used to upload data
-void iSENSE::set_email(string proj_email)
+// Users should never have to call this method, as it is possible to
+// pull datasets and compare dataset names to get the dataset_ID
+// Users should instead use the append byName method.
+void iSENSE::set_dataset_ID(string proj_dataset_ID)

```

Comment:
That makes sense. I'll try running the code with the method set as private and make sure everything works.